tests/integration/test_storage_s3_queue/test.py

```diff
@@ -3048,3 +3048,87 @@ def get_count():
     ) or node1.contains_in_log(
         f"StorageS3Queue (r.{table_name}): Skipping file {failed_file}: Failed"
     )
+
+
+def test_failed_commit(started_cluster):
+    node = started_cluster.instances["instance"]
+
+    table_name = f"test_failed_commit"
+    dst_table_name = f"{table_name}_dst"
+    keeper_path = f"/clickhouse/test_{table_name}_{generate_random_string()}"
+    files_path = f"{table_name}_data"
+    files_to_generate = 1
+
+    create_table(
+        started_cluster,
+        node,
+        table_name,
+        "unordered",
+        files_path,
+        additional_settings={
+            "keeper_path": keeper_path,
+        },
+    )
+    total_values = generate_random_files(
+        started_cluster, files_path, files_to_generate, start_ind=0, row_num=2
+    )
+
+    node.query(f"SYSTEM ENABLE FAILPOINT object_storage_queue_fail_commit")
+
+    create_mv(node, table_name, dst_table_name)
+
+    def check_failpoint():
+        return node.contains_in_log(
+            f"StorageS3Queue (default.{table_name}): Failed to process data: Code: 1002. DB::Exception: Failed to commit processed files. (UNKNOWN_EXCEPTION)"
+        )
+
+    for _ in range(100):
+        if check_failpoint():
+            break
+        time.sleep(50)
```

Comment:
50 seconds seems too much. Why can't we just increase the frequency of file processing?

Reply:
That's a typo, it must not be that big